### PR TITLE
committing slack webhook documentation

### PR DESCRIPTION
**Pull request by grabnerandi**
This is my PR to add documentation and information about how to integrate Slack via the Keptn Webhook Service. This approach was discussed on keptn slack channel with Johannes Bräuer

### CODE REVIEW

File: slack/0.10.0/README.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+# Slack Notifications via Keptn Webhooks
+
+This integration shows you how to send notifications to Slack leveraging [Keptn's Webhook Service](https://keptn.sh/docs/0.10.x/integrations/webhooks/).
```
Comment (oleg-nenashev) [inline slack/0.10.0/README.md:3]: IIRC we have no support for "latest" link in the documentation so there might be some misconnect

File: slack/0.10.0/README.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+[WATCH THIS VIDEO TUTORIAL](https://www.youtube.com/watch?v=0vJS7ecayGw&t=7s) to see this in action!
```
Comment (oleg-nenashev) [inline slack/0.10.0/README.md:5]: Not important: Could also be a thumbnail image with redirect in the bottom

**Review by oleg-nenashev — APPROVED** [review]
I have not tested the guide manually, but it looks great text-wise. +1 for merging

**Comment by johannes-b** [comment]
Many thanks for the PR!

I would like to edit the PR directly since I see it as a replacement of: https://keptn.sh/docs/0.10.x/integrations/webhooks/slack/

**Comment by johannes-b** [comment]
@oleg-nenashev I have done some changes to this PR (however in a separate one) in order to position this example as a reference template for other Webhook-based integrations. 

Can you please take a look here: #33

Once we have agreed on #33, I'm happy to merge this one.

**Comment by johannes-b** [comment]
After merging this PR, we should get rid of: https://keptn.sh/docs/0.10.x/integrations/webhooks/slack/

**Pull request merged**